### DISCUSSION TITLE
Is sorting slow in V8?

### DISCUSSION THREAD

**Discussion by vicke4** [TanStack/table#5058] [General] [answered]
I'm migrating from V7 to V8. The table uses column visibility, virtualized rows & sorting functionalities.

Everything works fine. But I noticed sorting is slow in V8.

Am I missing something? Has anybody noticed the same?

**Comment by sledgehammer999** [comment]
FYI, two related bug reports for [Svelte](https://github.com/TanStack/table/issues/4962) and [Solid](https://github.com/TanStack/table/issues/5019).

**Comment by vicke4** [comment, marked as answer]
I was using the wrong key on the `tr` tag. After fixing that, everything is working fine.
